packages/payment-integration-api/src/billing/is-billing-address-like.ts

```diff
@@ -0,0 +1,9 @@
+import { Address } from '../address';
+
+import BillingAddress from './billing-address';
+
+export default function isBillingAddressLike(address: Address): address is BillingAddress {
+    const billingAddress = address as BillingAddress;
+
+    return typeof billingAddress.id !== 'undefined';
+}
```

Comment:
It might be better to check all required billing address fields to make sure that it is a billing address. Also if its possible, try to avoid using 'as' operator.

Reply:
The only required field in BillingAddress is „id”, „email” is optional. So if this function is checking if provided address argument is type of BillingAddress, I think that checking „id” is sufficient.
In term of assertion looks like it is necessary here.